src/squishy/data_readers.clj

```diff
@@ -1,7 +1,16 @@
 (ns squishy.data-readers
+  (:require [clojure.string :as str])
   (:import [com.amazonaws.regions Region Regions]))
 
-(defn aws-region [region-name]
-  (-> region-name
-      (Regions/valueOf)
-      (Region/getRegion)))
+(defn aws-region
+  "Attempts to coerce the string into a Java Region instance. Can accept
```

Comment:
What it it doesn't coerce to a reasonable `Region`, such as `us-underwater-basket-weaving-1`?

Reply:
Yeah, I thought about that too, but the previous implementation was also brittle in terms of unexpected values. The only thing I could do is throw my own customized exception, but I don't know that it's any more valuable than letting the Java code do the same.